verification/epic.go

```diff
@@ -27,7 +27,7 @@ var VerifierPath string
 var ErrMissingSource = errors.New("MissingSource")
 var ErrVerifierStatus = errors.New("VerifierStatus")
 var ErrVideoUnavailable = errors.New("VideoUnavailable")
-var ErrAudioMismatch = Retryable{errors.New("AudioMismatch")}
+var ErrAudioMismatch = Fatal{Retryable{errors.New("AudioMismatch")}}
```

Comment:
Slightly out of scope of this PR as I realise (or is it realize? ) you're adhering to the structure that was previously in place, but unless simply logging a statement, errors should not be capitalised. 

https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
100% agreed.  Do you want to write a new ticket for this?  I agree that it should be done, but I can't speak to its priority.

Reply:
While we're at it, there are plenty of functions that `return errors.New("foo")` - we should declare those errors as global, and only once, and avoid calling errors.New() multiple times for the same error object.   Of course, that is also out of scope for this PR.